<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TargetManager</class>
 <widget class="QDialog" name="TargetManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>925</width>
    <height>752</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Target Manager</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Source</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QTabWidget" name="tabWidget">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="tabPosition">
          <enum>QTabWidget::West</enum>
         </property>
         <property name="currentIndex">
          <number>2</number>
         </property>
         <widget class="QWidget" name="csvTab">
          <attribute name="title">
           <string>CSV</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="0" column="0" colspan="2">
            <widget class="QGroupBox" name="csvGroup">
             <property name="title">
              <string>CSV file</string>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
             <layout class="QGridLayout" name="gridLayout_7">
              <item row="0" column="0" colspan="2">
               <widget class="QLineEdit" name="csvFilepath"/>
              </item>
              <item row="0" column="2">
               <widget class="QToolButton" name="browseForCSV">
                <property name="text">
                 <string>...</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QPushButton" name="addCSVTarget">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Add</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0" colspan="2">
            <widget class="QLineEdit" name="csvTargetFilter">
             <property name="text">
              <string/>
             </property>
             <property name="placeholderText">
              <string>Filter...</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="loadCSV">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Load</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="2" column="0" colspan="2">
            <widget class="QTreeView" name="csvTargets">
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
             <property name="rootIsDecorated">
              <bool>false</bool>
             </property>
             <property name="sortingEnabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="hipparcosTab">
          <attribute name="title">
           <string>Hipparcos</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0" colspan="2">
            <widget class="QGroupBox" name="hipparcos2Group">
             <property name="title">
              <string>Hipparcos 2 data file</string>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLineEdit" name="hipparcos2Filepath"/>
              </item>
              <item>
               <widget class="QToolButton" name="browseForHipparcos2">
                <property name="text">
                 <string>...</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="1" column="0" colspan="2">
            <widget class="QGroupBox" name="hipparcosGroup">
             <property name="title">
              <string>Optional original Hipparcos data file</string>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
             <layout class="QGridLayout" name="gridLayout_5">
              <item row="1" column="1">
               <widget class="QLineEdit" name="hipparcosFilepath"/>
              </item>
              <item row="1" column="2">
               <widget class="QToolButton" name="browseForHipparcos">
                <property name="text">
                 <string>...</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Used to associate the entries in the Hipparcos 2 data file with the names from the original Hipparcos data</string>
                </property>
                <property name="wordWrap">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QPushButton" name="loadHipparcos">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Load</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0" colspan="2">
            <widget class="QLineEdit" name="hipparcosTargetFilter">
             <property name="text">
              <string/>
             </property>
             <property name="placeholderText">
              <string>Filter...</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QPushButton" name="addHipparcosTarget">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Add</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="3" column="0" colspan="2">
            <widget class="QTreeView" name="hipparcosTargets">
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
             <property name="rootIsDecorated">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="simbadTab">
          <attribute name="title">
           <string>SIMBAD</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_8">
           <item row="1" column="0" colspan="3">
            <widget class="QLineEdit" name="simbadQuery">
             <property name="placeholderText">
              <string>Search...</string>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QPushButton" name="querySimbad">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>Go</string>
             </property>
            </widget>
           </item>
           <item row="4" column="3">
            <widget class="QPushButton" name="addSimbadTarget">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Add</string>
             </property>
             <property name="checkable">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="0" colspan="4">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Query the online astronomical database SIMBAD</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0" colspan="4">
            <widget class="QTreeView" name="simbadTargets">
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
             <property name="rootIsDecorated">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Targets</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QToolButton" name="loadTargets">
             <property name="text">
              <string>...</string>
             </property>
             <property name="icon">
              <iconset resource="../icons/icons.qrc">
               <normaloff>:/document-open.svg</normaloff>:/document-open.svg</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="saveTargets">
             <property name="text">
              <string>...</string>
             </property>
             <property name="icon">
              <iconset resource="../icons/icons.qrc">
               <normaloff>:/document-save.svg</normaloff>:/document-save.svg</iconset>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QToolButton" name="removeAllTargets">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>...</string>
             </property>
             <property name="icon">
              <iconset resource="../icons/icons.qrc">
               <normaloff>:/edit-clear.svg</normaloff>:/edit-clear.svg</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTreeView" name="targets">
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="rootIsDecorated">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <spacer name="horizontalSpacer_5">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QToolButton" name="removeTarget">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>...</string>
             </property>
             <property name="icon">
              <iconset resource="../icons/icons.qrc">
               <normaloff>:/list-remove.svg</normaloff>:/list-remove.svg</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QCommandLinkButton" name="returnButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Return</string>
     </property>
     <property name="icon">
      <iconset resource="../icons/icons.qrc">
       <normaloff>:/go-previous.svg</normaloff>:/go-previous.svg</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../icons/icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>returnButton</sender>
   <signal>clicked()</signal>
   <receiver>TargetManager</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>462</x>
     <y>722</y>
    </hint>
    <hint type="destinationlabel">
     <x>462</x>
     <y>375</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
